clients/roscpp/include/ros/service_client.h

```diff
@@ -160,6 +160,13 @@ class ROSCPP_DECL ServiceClient
    */
   std::string getService();
 
+  /**
+   * \brief Set timeout for service calls
+   * \param timeout The amount of time to wait for call response. If timeout is -1 (default),
```

Comment:
should probably mention that the unit is seconds

Reply:
Changed it to `ros::WallDuration`, so no need for that anymore.